src/amo/actions/reviews.js

```diff
@@ -1,33 +1,41 @@
 /* @flow */
 import invariant from 'invariant';
 
-import {
-  CLEAR_ADDON_REVIEWS,
-  FETCH_GROUPED_RATINGS,
-  FETCH_REVIEWS,
-  FETCH_USER_REVIEWS,
-  HIDE_EDIT_REVIEW_FORM,
-  HIDE_REPLY_TO_REVIEW_FORM,
-  SEND_REPLY_TO_REVIEW,
-  SEND_REVIEW_FLAG,
-  SET_ADDON_REVIEWS,
-  SET_GROUPED_RATINGS,
-  SET_INTERNAL_REVIEW,
-  SET_LATEST_REVIEW,
-  SET_REVIEW,
-  SET_REVIEW_REPLY,
-  SET_REVIEW_WAS_FLAGGED,
-  SET_USER_REVIEWS,
-  SHOW_EDIT_REVIEW_FORM,
-  SHOW_REPLY_TO_REVIEW_FORM,
-} from 'amo/constants';
 import type { FlagReviewReasonType } from 'amo/constants';
 import type {
   ExternalReviewReplyType,
   ExternalReviewType,
   GroupedRatingsType,
 } from 'amo/api/reviews';
 
+export const CLEAR_ADDON_REVIEWS: 'CLEAR_ADDON_REVIEWS' = 'CLEAR_ADDON_REVIEWS';
```

Comment:
This pattern seems nice, having the actions in `actions/reviews.js`, and only having "true" reducer code in the `reducers/reviews.js` file, but it's also inconsistent across the site. Is this an old pattern or a new pattern? I ask because it's not what we did with collections, and a lot of that is new code.

I'm not suggesting we go back and change a bunch of stuff just to be consistent, but it would be good to know, moving forward, what pattern we'd like to use, and it might be worth converting some things as we touch them, if it's not a huge amount of work.

Reply:
The new pattern is to put actions in the reducer module. The reviews code is pretty old but I didn't want to make the patch too crazy by moving it to the reducer. This is a good baby step, though.